apigee_m10n.module

```diff
@@ -40,6 +40,20 @@ function apigee_m10n_api_product_access(EntityInterface $entity, $operation, Acc
   return AccessResult::neutral();
 }
 
+/**
+ * Implements hook_apigee_edge_teams_team_api_product_access().
+ */
+function apigee_m10n_teams_apigee_edge_teams_team_api_product_access(ApiProductInterface $api_product, string $operation, TeamInterface $team, AccountInterface $account) {
```

Comment:
Hook name is not correct, you have added in m10n.module. I think this needs to be added in m10n teams module.

Reply:
Added in m10n module